pkg/operator/ceph/file/controller.go

```diff
@@ -270,3 +277,21 @@ func (c *FilesystemController) releaseOrchestrationLock() {
 	c.orchestrationMutex.Unlock()
 	logger.Debugf("Released lock for filesystem orchestration")
 }
+
+func updateCephFilesystemStatus(name, namespace, status string, context *clusterd.Context) {
+	updatedCephFilesystem, err := context.RookClientset.CephV1().CephFilesystems(namespace).Get(name, metav1.GetOptions{})
+	if err != nil {
+		logger.Errorf("Unable to update the cephObjectStore %s status %v", updatedCephFilesystem.GetName(), err)
+		return
+	}
+	if updatedCephFilesystem.Status == nil {
+		updatedCephFilesystem.Status = &cephv1.Status{}
+	} else if updatedCephFilesystem.Status.Phase == status {
+		return
+	}
+	updatedCephFilesystem.Status.Phase = status
+	_, err = context.RookClientset.CephV1().CephFilesystems(updatedCephFilesystem.Namespace).Update(updatedCephFilesystem)
+	if err != nil {
+		logger.Errorf("Unable to update the cephObjectStore %s status %v", updatedCephFilesystem.GetName(), err)
```

Comment:
For consistency let's also return after this error

Reply:
yup done.